src/app/app.component.css

```diff
@@ -55,14 +57,25 @@ height: 50px;
 
 .fa-list-ol
 {
+  margin-left: 8px;
+  margin-right: 5px;
   color: #F69736;
 }
 
 .fa-home
 {
+  margin-left: 8px;
+  margin-right: 5px;
   color: #F69736;
 }
 .fa-cloud-upload
 {
+  margin-left: 8px;
+  margin-right: 5px;
   color: #F69736;
 }
+
+.app-content
+{
+  height: 100vh;
```

Comment:
At last! Good point! We'll have to get this a little bit more reasonably in the near future, but for now it's good.